setup.py

```diff
@@ -114,11 +114,11 @@ def get_git_sha() -> str:
         "redis",
         "selenium>=3.141.0",
         "simplejson>=3.15.0",
-        "slackclient==2.5.0",  # PINNED! slack changes file upload api in the future versions
+        "slack_sdk>=3.1.1",
         "sqlalchemy>=1.4, <2",
         "sqlalchemy-utils>=0.38.3, <0.39",
         "sqlparse==0.3.0",  # PINNED! see https://github.com/andialbrecht/sqlparse/issues/562
-        "tabulate==0.8.9",
+        "tabulate>=0.8.9",
```

Comment:
..and here something similar, assuming typical semver?
```suggestion
        "tabulate>=0.8.9, <0.9",
```